### PR TITLE
 updated go source url to philipgriggs fork

### PR DESCRIPTION
**Pull request by philipgriggs**

### CODE REVIEW

**Comment by xylo** [comment]
Are you sure you want to link the web template files to your master repository?

I guess that could get complicated when you pull changes from the origin.  But I'm not sure.

**Comment by Dominik-K** [comment]
@philipgriggs I think it's better if you point to a new repository which just contains the Go  `.postfixTemplates` files. That will ease contributing to it, I guess. The Java part remains in your `intellij-postfix-templates` fork.

@xylo It would then makes sense to only have @philipgriggs's new repo as the `master` for the Go template files. I.e. removing the `./templates/go` from here.

**Comment by philipgriggs** [comment]
Ok, I understand. I've created my own repo and removed the go templates from the original repo

**Comment by Dominik-K** [comment]
@philipgriggs 

I suggest using 

* "completions" instead "templates" and
* "goland"
 
in the repo name, e.g. `goland-postfix-completions`. 

The name "postfix templates" is replaced with "postfix completions" ("Editor > General > Postfix Completions"), even in [IntelliJ](https://www.jetbrains.com/help/idea/settings-postfix-completion.html) (at least since 2016.3). @xylo Is there a reason for using "templates" in the name?

The completions will be mainly used for GoLand (didn't try it with Go Plugin in IntelliJ). To best attract contributions, help the search engines with "goland" in the name.

File: templates/webTemplateFiles.yaml [hunk 1]
```diff
@@ -126,42 +126,42 @@
   description: Provides templates for miscellaneous use cases
 - lang: go
   id: xylo.numbers
```
Comment (Dominik-K) [inline templates/webTemplateFiles.yaml]: Rename the IDs.

File: templates/webTemplateFiles.yaml [hunk 2]
```diff
@@ -126,42 +126,42 @@
...
-  url: https://raw.githubusercontent.com/xylo/intellij-postfix-templates/master/templates/go/numbers.postfixTemplates
+  url: https://raw.githubusercontent.com/philipgriggs/go-postfix-templates/master/numbers.postfixTemplates
   author: Philip Griggs
   email: philipgriggs@gmail.com
   website: https://github.com/xylo/intellij-postfix-templates/wiki/Go-web-templates:-xylo.numbers
```
Comment (Dominik-K) [inline templates/webTemplateFiles.yaml]: The wiki entries aren't there, yet. Easiest just link to your new repo and document the usage in the Readme.

**Comment by xylo** [comment]
@Dominik-K  Yes, the word `templates` is used by intension.  When JetBrains first introduced the postfix completion in IDEA it was not adaptable at all.  All postfix completions were hard coded.  I really liked the idea of the postfix completion but it was nearly useless because there were only these few hard coded completions.  In contrast to that IDEA had already a similar and very extensible solution for a context insensitive completion, called `live templates`.  So I adopted the IDEA of the live templates and brought it to the postfix completion.  Since the `live templates` were never called `live completion` or something like that I also chose to skip the word `completion` for the postfix templates.

However, I have to admit that the words `custom` and `templates` are redundant.  I did this mostly for better search results.  And I have to admit that I probably have better named it `custom postfix completion templates` for even better search results.  Anyway, that's the name since some years and I don't know if it's still useful to change it.

File: templates/webTemplateFiles.yaml (hunk 1 above)
Comment (xylo) [reply templates/webTemplateFiles.yaml]: Yeah, I created the same review some days ago but forgot to submit it. :(
@philipgriggs Can you change the IDs?  Maybe to something like philipgriggs.numbers or shorter.

File: templates/webTemplateFiles.yaml (hunk 2 above)
Comment (xylo) [reply templates/webTemplateFiles.yaml]: Yes. The wiki links should also point to the new repo.

**Comment by xylo** [comment]
@Dominik-K The Go templates are not only for Goland.  They can be used in any JetBrains release that supports Go, e.g. Intellij IDEA.

**Comment by xylo** [comment]
Thanks for addressing the issues.

**Pull request merged**